### PR TITLE
Feature/2 dc configuration dialog

### LINKED ISSUE #5: PDI should use the centrally registered DC installation
PDI should use the centrally registered DC installation

### PR DESCRIPTION
**Pull request by ClaudiaPHI**
Fixes #5

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -125,30 +126,7 @@
...
-					<execution>
-						<id>enforce-versions</id>
-						<goals>
-							<goal>enforce</goal>
-						</goals>
-						<configuration>
-							<rules>
-								<bannedDependencies>
-									<excludes>
-										<exclude>xml-apis:xml-apis</exclude>
```
Comment (kaspersorensen) [inline pom.xml:144]: Why was this removed?

File: src/main/java/org/pentaho/di/profiling/datacleaner/ModelerHelper.java [hunk 2]
```diff
@@ -118,6 +124,25 @@ public String getName() {
...
+    private static String getDataCleanerInstalationPath(String pluginFolderPath) {
+        final String configurationFilePath = pluginFolderPath + "/" + DATACLEANER_CONFIG_FILE;
+
+        try {
+            final BufferedReader inputReader = new BufferedReader(new FileReader(configurationFilePath));
+            final String dcInstallationPath = inputReader.readLine().trim();
+            inputReader.close();
+            return dcInstallationPath;
+        } catch (Exception e) {
+            new ErrorDialog(
```
Comment (kaspersorensen) [inline src/main/java/org/pentaho/di/profiling/datacleaner/ModelerHelper.java]: Just wondering/asking: Shouldn't one call _something_ on a dialog to show it? Here it is just instantiated, but I could imagine it having a ".show()" or ".setVisible(true)" like method?

**Comment by kaspersorensen** [comment]
LPGTMEFTR (Looks Pretty Good To Me Except For The Remarks) ;)

File: pom.xml (hunk 1 above)
Comment (ClaudiaPHI) [inline pom.xml:144]: I have got an error  of the sort ' I can't exclude xml-api because I cannot find it '. So I removed it.

File: src/main/java/org/pentaho/di/profiling/datacleaner/ModelerHelper.java (hunk 2 above)
Comment (ClaudiaPHI) [inline src/main/java/org/pentaho/di/profiling/datacleaner/ModelerHelper.java]: There is no need for it for a show method.

**Comment by ClaudiaPHI** [comment]
I show repeat first grade again so that I learn to read.

**Comment by kaspersorensen** [comment]
Why do we have an empty datacleaner-configuration.xml file? Not that it hurts, but I guess we could just create one on the fly if content is not needed.

**Comment by ClaudiaPHI** [comment]
The datacleaner-configuration.xml contains the installation path of the local DataCleaner. 
If it does not exit, one is created automatically. I can delete it.

**Comment by ClaudiaPHI** [comment]
I think it's ready for merging
